dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -64,14 +65,28 @@ HRESULT CFSDropTarget::_CopyItems(IShellFolder * pSFFrom, UINT cidl,
     if (FAILED(ret))
         goto cleanup;
 
-    wszDstPath[lstrlenW(wszDstPath) + 1] = L'\0';
+    wszDstPath[lstrlenW(wszDstPath) + 1] = UNICODE_NULL;
+
+    /* Set TRUE to bRenameOnCollision if necesssary */
+    if (bCopy)
+    {
+        WCHAR szPath1[MAX_PATH], szPath2[MAX_PATH];
+        GetFullPathNameW(pwszSrcPathsList, _countof(szPath1), szPath1, NULL);
+        GetFullPathNameW(wszDstPath, _countof(szPath2), szPath2, NULL);
+        PathRemoveFileSpecW(szPath1);
```

Comment:
I don't remember whether this has already been a past question: `szPath2` doesn't need to have a PathRemoveFileSpecW call on it too?

Reply:
`wszDstPath` is already a directory.